public/auth-failure.html

```diff
@@ -0,0 +1 @@
+<p>Oh no!</p>
```

Comment:
I wasn't able to reach this page by entering the wrong credentials. I reached a Twitter page instead.
How can we reach this page?